Kwartracker/Scenes/Reusable/ImageView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ImageView.swift
+//  Kwartracker
+//
+//  Created by Veverly Mae Ricaza Veverly on 4/29/21.
+//
+
+import SwiftUI
+
+struct ImageView: View {
+    
+    let imageType: ImageType
+    
+    var body: some View {
+        VStack{
+            Image(uiImage: imageType.image)
+        }
+    }
+    
+    enum ImageType {
```

Comment:
Actually all of these enums with success and delete can be refactored into just one enum, maybe DialogState? then you'll have two options, either make the message included in the cases or create another params per view with an additional param `message`